lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -2004,7 +2004,7 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :description => 'If `true`, the agent captures [custom events](/docs/insights/new-relic-insights/adding-querying-data/inserting-custom-events-new-relic-apm-agents).'
         },
         :'custom_insights_events.max_samples_stored' => {
-          :default => 1000,
+          :default => 3000,
```

Comment:
Is there any impact to increasing this number? I'm wondering how we can up with 1000 in the first place, and why 3000 is the new number.

Reply:
Hi Hannah! Please [see this slack thread (internal)](https://newrelic.slack.com/archives/GAPN6K7T2/p1665607818802639?thread_ts=1665607325.560299&cid=GAPN6K7T2)

1000 is actually lower than what all the other agents have at this time. The spec default is 10K per limit, so we're getting closer to the spec with a smaller performance impact by changing the value to 3K.